.travis.yml

```diff
@@ -2,18 +2,17 @@ language: c
 sudo: required
 install:
   - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travis-coveralls/master/travis-coveralls.sh
 script:
   - bash -ex .travis-opam.sh
-  - if [ ! -z "${XS_COV}" ]; then bash -ex travis-coveralls.sh || true; fi
 env:
   global:
     - OCAML_VERSION=4.04
```

Comment:
I guess we should update this to Docker in a future PR.

Reply:
Until `docker-opam` is fixed to properly support `BASE_REPO` I cannot update it